src/main/java/thrift/model/ModelManager.java

```diff
@@ -136,11 +148,6 @@ public void setTransaction(Transaction target, Transaction updatedTransaction) {
         thrift.setTransaction(target, updatedTransaction);
     }
 
-    @Override
-    public Transaction getLastTransactionFromThrift() {
-        return thrift.getLastTransaction();
```

Comment:
Is there a case where we need to retrieve the last `Transaction`?

Reply:
Resolved at Commit afe1d05.